contrib/format-image/pom.xml

```diff
@@ -39,7 +39,7 @@
     <dependency>
       <groupId>com.drewnoakes</groupId>
       <artifactId>metadata-extractor</artifactId>
-      <version>2.18.0</version>
+      <version>2.19.0</version>
```

Comment:
I guess there's a bug that got fixed in this library and that's now breaking a unit test?

Reply:
Now that I've seen DRILL-8472, did you mean to include [0fda54c](https://github.com/apache/drill/pull/2862/commits/0fda54c2432efc91181ed19af5af0122f46d8dbd) in this PR?

Reply:
Yes, but I have to fix one unit test.

Reply:
@jnturton I fixed this.